server/service/src/processors/transfer/requisition/create_response_requisition.rs

```diff
@@ -46,15 +50,48 @@ impl RequisitionTransferProcessor for CreateResponseRequisitionProcessor {
         } = &record_for_processing;
         // 2.
         if request_requisition.requisition_row.r#type != RequisitionType::Request {
-            return Ok(None);
+            return Ok(RequisitionTransferOutput::NotRequest);
         }
         // 3.
         if request_requisition.requisition_row.status != RequisitionStatus::Sent {
-            return Ok(None);
+            return Ok(RequisitionTransferOutput::NotSent);
         }
         // 4.
         if response_requisition.is_some() {
-            return Ok(None);
+            return Ok(RequisitionTransferOutput::HasResponse);
+        }
+        // 5.
+        if let Some(sent_datetime) = request_requisition.requisition_row.sent_datetime {
+            let pref_months = PreventTransfersMonthsBeforeInitialisation {}
+                .load(connection, None)
+                .map_err(|e| RepositoryError::DBError {
+                    msg: e.to_string(),
+                    extra: "".to_string(),
+                })?;
+            if pref_months > 0 {
+                let sort = Sort {
+                    key: SyncLogSortField::DoneDatetime,
+                    desc: None,
+                };
+
+                let latest_log_sorted_by_finished_datetime = SyncLogRepository::new(connection)
```

Comment:
is this actually `first_fully_processed_log`? `latest` makes it sound like most recent, but I think that's not right if we're trying to find initialisation date?

Reply:
That's completely fair. In my defence I copy pasted it 😁